src/tests/Common/Coreclr.TestWrapper/MobileAppHandler.cs

```diff
@@ -58,7 +58,7 @@ private static void HandleMobileApp(string action, string platform, string categ
                 appExtension = "app";
             }
 
-            cmdStr = $"{dotnetCmd} {xharnessCmd} {platform} {action} --package-name=net.dot.{category} --app={testBinaryBase}/{category}.{appExtension} --output-directory={reportBase}/install";
```

Comment:
This is the change which caused Android apps failed to install. The app installation error message is:

> XHarness command issued: android install --package-name=net.dot.JIT_Intrinsics --output-directory=/datadisks/disk1/work/B5720A12/w/B35609BF/uploads/Reports/install
[19:18:38] fail: Required argument app|a= was not supplied
XHarness exit code: 3 (INVALID_ARGUMENTS)

Reply:
@fanyang-mono I was removing the `--app` argument because it is no longer accepted where it's not needed (i.e. the `run` command).

However, in the logs I've posted, I don't see any attempt to install the app:
https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-runtime-refs-pull-54543-merge-d33e7953be674ec7bf/JIT/console.12c4ea14.log?sv=2019-07-07&se=2021-07-13T19%3A04%3A55Z&sr=c&sp=rl&sig=LHLZIFt4KSWxPGpSReQuCeqds6Vnfqotj4RcUE2ug00%3D

Reply:
I pushed a naive fix to unblock this PR. The `MobileAppHandler` will need to be changed later anyway because as is won't work for Apple for example. So let's see